test/parallel/test-timers-api-refs.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+// don't verify the globals for this test
+common.globalCheck = false;
+
+// try overriding global APIs to make sure
+// they're not relied on by the timers
+global.clearTimeout = assert.fail;
```

Comment:
You can't access 3 of the six. They're on the public API of the module, but that's not available to user code like tests.